### DISCUSSION TITLE
Satranç turnuvası sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#120] [Zeka Sorusu]
Sitemizde düzenlenen her oyuncunun diğer oyuncularla bir kez oynadığı bir satranç turnuvasında toplam 300 oyun oynanmıştır.

Turnuvada kaç satranççı vardır?

**Comment by Sevvalgocgun1** [comment]
25

**Reply by sonsuzus** [reply]
Açıklamalı çözüyoruz.
